packages/mini-cart/package.json

```
@@ -0,0 +1,61 @@
+{
+	"name": "@automattic/mini-cart",
+	"version": "1.0.0",
+	"description": "Functions and components used by WordPress.com checkout",
```

It appears we need to update this description. 
```suggestion
	"description": "A simple wp.com shopping-cart interface.",
```